### PR TITLE
Adds optional num_queues parameter to ocl_env_init()

### PR DESCRIPTION
**Pull request by sergisiso**

### CODE REVIEW

**Comment by sergisiso** [comment]
@arporter Can you review this short PR? It adds an optional parameter to `ocl_env_init` to provide a user-specified number of OpenCL queues.

File: src/FortCL.f90 [hunk 1]
```diff
@@ -37,18 +37,23 @@ module fortcl
...
 
-    if(cl_env_initialised)return
+    if(cl_env_initialised) return
 
     ! Initialise the OpenCL device
     call init_device(cl_device, cl_version_str, cl_context)
 
     ! Create command queue(s)
-    cl_num_queues = 4
+    if (.not.present(num_queues)) then
```
Comment (arporter) [inline src/FortCL.f90:52]: Sorry to be picky but please could you add a check that num_queues is > 1 if it is present. Apart from that it's fine.

**Review by arporter — CHANGES_REQUESTED** [review]
Just needs a slight tweak.

File: src/FortCL.f90 (hunk 1 above)
Comment (sergisiso) [reply src/FortCL.f90:52]: Added check.

**Comment by sergisiso** [comment]
@arporter Added the requested check. Ready for next review.

**Review by arporter — APPROVED** [review]

**Pull request merged**